recipes/abacat/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "abacat" %}
+{% set version = "0.0.4a" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/vinisalazar/abacat/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: ad36c110f35995970f4c33efb844e7dd8bc718498b0ede6bdd7032105ee47227
+
+build:
+  number: 1
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  noarch: python
+  run_exports:
+    - {{ pin_subpackage("abacat", max_pin="x.x") }
+    
+    
+requirements:
+  build:
+    - python >=3.6
+  run:
+    - python >=3.6
+
+test:
+  commands:
+    - abacat
+
+about:
+  home: https://github.com/vinisalazar/abacat
+  license: GPL-3
+  license_family: GPL
+  license_file: LICENSE.md
+  summary: 'abacat'
+  description: |
+    Abacat (pronounced "ABBA-cat") is a toolkit for working with bacterial whole genome sequencing (WGS) data. 
+    It provides Python objects to represent elements which are common to WGS analysis workflows, 
+    such as coding sequence (CDS) files, containing genes or proteins, alignment methods, or statistics about your sequences.
+    It can be used to annotate freshly generated WGS data and parse existing, curated NCBI data. 
+    It can also index different BLAST or HMM databases for easy alignment searches and produce comparisons using Average Nucleotide Identity (ANI) measurements. 
+    Lastly, Abacat's classes provides means of tracking data provenance in large scale genomics experiments, in a way users can monitor and tailor their workflow accordingly.
+extra:
+  additional-platforms:
+    - linux-aarch64
```

Comment:
The package is `noarch` - no need of `linux-aarch64`

Reply:
Let's wait for https://github.com/bioconda/bioconda-recipes/pull/46425#issuecomment-1999101539